--- Share/Lib.WCheck.php.orig	2008-12-01 15:44:35.000000000 +0100
+++ Share/Lib.WCheck.php	2008-12-01 16:04:59.000000000 +0100
@@ -280,7 +280,7 @@
   $tact[] = "$pubdir/wstart";
   global $_SERVER;
   if ($_SERVER['HTTP_HOST'] != "")  $tact[] = "sudo $pubdir/admin/shttpd";
-  else $tact[] = "service httpd restart";  
+  else $tact[] = "/etc/init.d/apache2 restart";  
 }
 
 function cmpapp($a,$b) {
--- Admin/shttpd.orig	2008-12-01 16:06:27.000000000 +0100
+++ Admin/shttpd	2008-12-01 16:14:51.000000000 +0100
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-echo 'service httpd restart' | at now + 1minutes
-echo "httpd will be restarted in one minute"
+echo '/etc/init.d/apache2 restart' | at now + 1minutes
+echo "apache2 will be restarted in one minute"

