--- What/Share/Lib.WCheck.php.orig	2007-07-04 15:25:27.000000000 +0200
+++ What/Share/Lib.WCheck.php	2007-07-04 15:25:41.000000000 +0200
@@ -291,7 +291,7 @@
   $tact[] = "$pubdir/wstart";
   global $_SERVER;
   if ($_SERVER['HTTP_HOST'] != "")  $tact[] = "sudo $pubdir/admin/shttpd";
-  else $tact[] = "/etc/rc.d/init.d/httpd restart";
+  else $tact[] = "/etc/init.d/apache2 restart";
   
 }
 
@@ -305,4 +305,4 @@
   if (isset($b["iorder"])) return 1;
   return 0;
 }
-?>
\ Pas de fin de ligne à la fin du fichier.
+?>
